example/src/components/OperationsSection/OperationsSection.js

```diff
@@ -618,6 +621,36 @@ const OperationsSection = () => {
               isAnimating={sendFilledAnimating} 
               size={32} />
           </li> 
+          <li
+            className='icon-tile'
+            onMouseEnter={() => setSortAscendingAnimating(true)}
+            onMouseLeave={() => setSortAscendingAnimating(false)}
+          >
+            <h3>Sort ascending</h3>
```

Comment:
Feel like you should consider creating an `IconTile` component so if you ever need to change the structure, you won’t need to replace all these `<h3>`s. Probably feels pretty minor for now, but could come in handy down the road if you only need to make the changes in one place instead of for each instance.

Reply:
I agree 110%! It's something I've planned on fixing but haven't gotten to yet.  I thought I had an issue open for it but I didn't, so I'll make one now!